tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/conflicting_hpo_state_generalization_test.py

```diff
@@ -0,0 +1,112 @@
+"""
+ Integration test for generalization of conflicting HPO States.
+
+ Original Issue: DC-512
+ """
+# Python imports
+import os
+
+# Third Party Imports
+
+# Project Imports
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.deid.conflicting_hpo_state_generalization import ConflictingHpoStateGeneralize
+from common import JINJA_ENV, OBSERVATION, VOCABULARY_TABLES
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+
+INSERT_RAW_DATA_OBS = JINJA_ENV.from_string("""
+   INSERT INTO `{{project_id}}.{{dataset_id}}.observation` (
+       observation_id,
+       person_id,
+       observation_concept_id,
+       observation_date,
+       observation_type_concept_id
+       )
+     VALUES
+       (1,101,1001,date('2020-01-01'),1),
+       (2,102,1002,date('2020-01-01'),1),
+       (3,103,1003,date('2020-01-01'),1),
+       (4,104,1004,date('2020-01-01'),1)
```

Comment:
The API has gotten better since we started the project.  You no longer have to wrap the date strings in `date()` object.  You can update this if you want.